src/index.js

```diff
@@ -0,0 +1,14 @@
+import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.classList.add('hello');
+
+  return element;
+}
```

Comment:
- [ ] One of the project's requirements is to use ES6 syntaxes. Kindly use arrow functions instead of the normal function definition. :pray:

Reply:
Resolved